tests/test_tuners_utils.py

```diff
@@ -55,14 +55,16 @@
     ("foo.bar.1.baz", ["baz"], [0, 1, 2], ["bar"], True),
     ("foo.bar.1.baz", ["baz", "spam"], [1], ["bar"], True),
     ("foo.bar.1.baz", ["baz", "spam"], [0, 1, 2], ["bar"], True),
-    # TODO: Unclear what expected behaviour is when layers_pattern is an empty list.
-    # Currently, an empty layers_pattern leads to all layer indexes being matched,
-    # which means layers_to_transform is ignored.
-    ("foo.bar.1.baz", ["baz"], [1], [], True),
-    # TODO: Below test currently fails, again because of empty layers_pattern
-    # layers_to_transform is 0, but layers_pattern is empty, so all layer indexes are matched
-    # ("foo.bar.1.baz", ["baz"], [0], [], False),
-    ("foo.bar.1.baz", ["baz"], [1], ["ar"], True),
+    # empty layers_to_transform
+    ("foo.bar.7.baz", ["baz"], [], ["bar"], True),
+    ("foo.bar.7.baz", ["baz"], None, ["bar"], True),
```

Comment:
Also test `""` for `layers_to_transform`?

Reply:
Due to the type of this arg `layers_to_transform: Optional[Union[List[int], int]])`, these two should be enough.